src/functionalTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ProcessedEnvelopeMessageHandlingTest.java

```diff
@@ -112,11 +109,18 @@ private EnvelopeResponse getEnvelope(String zipFilename) {
             );
     }
 
+    //unknown fields should be ignored
     private void sendProcessedEnvelopeMessage(UUID envelopeId) throws Exception {
         IMessage message = new Message(
-            objectMapper.writeValueAsString(new ProcessedEnvelope(envelopeId))
+            " {"
+                + "\"id\":\"" + envelopeId + "\","
+                + "\"processedCcdReference\":\"ccd-ref-9283\","
+                + "\"processedCcdType\":\"ccd-type-exception\","
```

Comment:
are those 2 invalid on purpose?

Reply:
these 2 are new fields

Reply:
the new fields use underscores

Reply:
ie it's `"processed_ccd_type"` as opposed to `"processedCcdType"`